requirements.txt

```diff
@@ -0,0 +1,4 @@
+pytest
+mock
+websocket
```

Comment:
We don't depend on `websocket` either, only `websocket-client`.

Reply:
👍